### PR TITLE
InfluxDB 2.7.4

### PR DESCRIPTION
**Pull request by sanderson**
- Updates InfluxDB patch version to 2.7.4.
- Fixes `latest-patch` shortcode to correctly get the latest `influx` CLI version
- Update all downloads page links

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/telegraf/v1/install.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Examples are installed at `/usr/local/etc/telegraf.conf.sample`.
 {{% tab-content %}}
 Choose from the following options to install Telegraf for macOS:
 
-- To manually install Telegraf from a file, see the [downloads page](https://portal.influxdata.com/downloads/).
+- To manually install Telegraf from a file, see the [downloads page](https://www.influxdata.com/downloads//).
```
Comment (jstirnaman) [inline content/telegraf/v1/install.md:170]:
```suggestion
- To manually install Telegraf from a file, see the [downloads page](https://www.influxdata.com/downloads/).
```

File: content/telegraf/v1/install.md [hunk 2]
```diff
@@ -176,11 +176,11 @@ Users of macOS 10.8 and higher can install Telegraf using the [Homebrew](http://
 
 {{% note %}}
 
-The `telegraf` binary installed by Homebrew differs from the macOS `.dmg` builds available from the [downloads page](https://portal.influxdata.com/downloads/).
+The `telegraf` binary installed by Homebrew differs from the macOS `.dmg` builds available from the [downloads page](https://www.influxdata.com/downloads//).
```
Comment (jstirnaman) [inline content/telegraf/v1/install.md:179]:
```suggestion
The `telegraf` binary installed by Homebrew differs from the macOS `.dmg` builds available from the [downloads page](https://www.influxdata.com/downloads/).
```

File: content/influxdb/v2/install/_index.md [hunk 3]
```diff
@@ -288,7 +288,7 @@ For information about installing the `influx` CLI, see
 ### Install InfluxDB as a service with systemd
 
 1.  Download and install the appropriate `.deb` or `.rpm` file using a URL from the
-    [InfluxData downloads page](https://portal.influxdata.com/downloads/)
+    [InfluxData downloads page](https://www.influxdata.com/downloads//)
```
Comment (jstirnaman) [inline content/influxdb/v2/install/_index.md:291]:
```suggestion
    [InfluxData downloads page](https://www.influxdata.com/downloads/)
```

File: content/influxdb/v1/administration/upgrading.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ and see [Migrate to InfluxDB Enterprise](/enterprise_influxdb/v1/guides/migratio
 
 ## Upgrade to InfluxDB 1.8.x
 
-1. [Download](https://portal.influxdata.com/downloads) InfluxDB version 1.8.x and [install the upgrade](/influxdb/v1/introduction/installation).
+1. [Download](https://www.influxdata.com/downloads/) InfluxDB version 1.8.x and [install the upgrade](/influxdb/v1/introduction/installation).
```
Comment (jstirnaman) [inline content/influxdb/v1/administration/upgrading.md:28]:
```suggestion
1. [Download](https://www.influxdata.com/downloads/) InfluxDB version 1.8.x and [install the upgrade](/influxdb/v1/introduction/installation/).
```

File: content/influxdb/clustered/reference/release-notes/influxctl.md [hunk 5]
```diff
@@ -175,7 +175,7 @@ following guidelines:
 ### Features
 
 - Add support for both InfluxDB Cloud Dedicated and InfluxDB Clustered.
-- Provide public distributions through <https://portal.influxdata.com/downloads/>
+- Provide public distributions through <https://www.influxdata.com/downloads//>
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/release-notes/influxctl.md:178]:
```suggestion
- Provide public distributions through <https://www.influxdata.com/downloads/>
```

File: content/influxdb/cloud/upgrade/v1-to-cloud/_index.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ for the upgrade process.
 _For more information about managing tokens and token types, see [Manage tokens](/influxdb/cloud/admin/tokens/)._
 
 ## Download and install the influx CLI
-1.  Visit the [InfluxDB downloads page](https://portal.influxdata.com/downloads/)
+1.  Visit the [InfluxDB downloads page](https://www.influxdata.com/downloads//)
```
Comment (jstirnaman) [inline content/influxdb/cloud/upgrade/v1-to-cloud/_index.md:46]:
```suggestion
1.  Visit the [InfluxDB downloads page](https://www.influxdata.com/downloads/)
```

File: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md [hunk 7]
```diff
@@ -174,7 +174,7 @@ following guidelines:
 ### Features
 
 - Add support for both InfluxDB Cloud Dedicated and InfluxDB Clustered.
-- Provide public distributions through <https://portal.influxdata.com/downloads/>
+- Provide public distributions through <https://www.influxdata.com/downloads//>
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md:177]:
```suggestion
- Provide public distributions through <https://www.influxdata.com/downloads/>
```

**Comment by sanderson** [comment]
🤦🏻‍♂️ Thanks @jstirnaman. Nice catches.

**Pull request merged**